docs/guides/api-gateway/kubernetes.mdx

```diff
@@ -0,0 +1,414 @@
+traffic into your cluster's internal network: the original [Ingress
+Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+or the [Gateway
+API](https://kubernetes.io/docs/concepts/services-networking/gateway/).
+
+The [ngrok Kubernetes Operator](https://github.com/ngrok/ngrok-operator)
+supports both mechanisms. While we encourage you to explore both, we recommend
+the Gateway API based on the following:
+
+- The [Ingress
+  API](https://kubernetes.io/docs/concepts/services-networking/ingress/), which
+  Ingress Controllers utilize, is frozen in new versions of K8s and no longer
+  receives updates.
+- While Ingress Controller is designed purely for ingress, the Gateway API is
+  also designed for dynamic infrastructure provisioning and advanced traffic routing.
+- The Gateway API is role-oriented, allowing DevOps/platform/infrastructure
+  engineering teams to control clusters, policies, and permissions centrally,
+  while still allowing API developers to self-service endpoints to their services
+  with route objects.
+- The Gateway API provides a common interface to K8s networking, removing
+  reliance on vendor-specific configurations.
+- Many developers can integrate their APIs or apps onto a single `GatewayClass`
+  or `Gateway`, using `HTTPRoute`s, without affecting others.
+
+For the scope of this guide, and based on this recommendation, we'll focus on
+Gateway API.
+
+## **Step 0**. Deploy the example ngrok API service
+
+If you're building a proof of concept with ngrok as the API gateway for your
+APIs in Kubernetes, you can use our [example ngrok API
+service](https://github.com/ngrok-samples/ngrok-api-service). This service
+responds to all routes with information about the connection&mdash;think of it
+as the `pong` to your `curl`.
+
+:::note
+If you already have an API service, you can skip this step&mdash;just be aware
+that you'll need to edit the example configurations below to properly configure
+the ngrok Kubernetes Operator.
+:::
+
+To deploy the ngrok API service to your cluster, create the `prod` namespace and
+apply its `deployment.yaml` file.
+
+```bash
+kubectl create namespace prod
+kubectl apply -f https://raw.githubusercontent.com/ngrok-samples/ngrok-api-service/refs/heads/main/kubernetes/deployment.yaml
+```
+
+## **Step 1**. Apply the ngrok Kubernetes Operator {#apply-kubernetes-operator}
+
+Add the [ngrok Kubernetes Operator](https://github.com/ngrok/ngrok-operator)
+repo to Helm.
+
+```bash
+helm repo add ngrok https://charts.ngrok.com
+helm repo update
+```
+
+Apply the latest Gateway API CRDs to your cluster.
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
+```
+
+Set up a few variables to help apply the ngrok Kubernetes Operator.
+
+- `{AUTHTOKEN}`: Your ngrok authtoken, which you can find in your [ngrok
+  dashboard](https://dashboard.ngrok.com/get-started/your-authtoken).
```

Comment:
Is it this [page](https://dashboard.ngrok.com/get-started/your-authtoken?__hstc=159041573.809078f985fa555684f1553b9e8a3bdc.1729089601880.1729089601880.1729101803219.2&__hssc=159041573.1.1729101803219&__hsfp=2997760446) or [this page](https://dashboard.ngrok.com/authtokens)? I always use the latter.

Reply:
I always use the former! I just use the same authtoken for everything 🤷 

But I added mention to both?